packages/terra-form-select/src/_Option.jsx

```diff
@@ -68,17 +68,23 @@ const Option = ({
   ]);
 
   return (
+    // Voiceover is having issues in a multiselect dropdown where once it reads the first
+    // select option that has aria-selected, then it is unable to read any other Options
+    // in the dropdown. It will just repeat the last selected element for any new menu
+    // items selected
+    /* eslint-disable jsx-a11y/role-has-required-aria-props */
     <li
       {...customProps}
       role="option"
       disabled={disabled}
       className={optionClassNames}
-      aria-selected={isSelected}
+      aria-checked={isSelected}
       aria-disabled={disabled}
     >
       {(isCheckable || isAddOption) && <span className={cx('icon')} />}
       <span className={cx('display')}>{display}</span>
     </li>
```

Comment:
One thing to double check is whether or not these options are announced when they are contained within an OptGroup. When I initially investigated this issue nested lists had trouble with Voice Over.